api/src/main/java/org/pmiops/workbench/reporting/ReportingJobResult.java

```diff
@@ -2,7 +2,6 @@
 
 public enum ReportingJobResult {
```

Comment:
Do we use this enum for anything? Recovering from bad states or is that coming in a future PR?

Reply:
Not yet. It's either coming out, or will be incorporated into a more sophisticated return type. I haven't decided whether to roll up exceptions and keep going or not. But I have an error handling/correcting story in the backlog.